### PR TITLE
Feature curly braces

### PR DESCRIPTION
**Pull request by deepak-brahmania**
Feature - Curly Braces
## Description

Added Curly Braces in rules section.
## Related Issue

NA
## How Has This Been Tested?

Locally
## Tested on:

- [x] Desktop Chrome.
- [x] Desktop Safari.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated (any) documentation accordingly.

### CODE REVIEW

File: asset/css/style.css [hunk 1]
```diff
@@ -1,3 +1,31 @@
+body {
```
Comment (awebdeveloper) [inline asset/css/style.css]: y this .... y css

File: .stylelintrc.json [hunk 2]
```diff
@@ -13,14 +13,13 @@
     "selector-attribute-operator-space-before": "always",
     "selector-attribute-operator-space-after": "always",
     "declaration-block-trailing-semicolon": "always",
-    "declaration-no-important": true,
     "declaration-colon-space-before": "never",
     "declaration-colon-space-after": "always",
-    "property-no-vendor-prefix": true,
+    "property-no-vendor-prefix": null,
```
Comment (awebdeveloper) [inline .stylelintrc.json]: y

File: .stylelintrc.json [hunk 3]
```diff
@@ -13,14 +13,13 @@
...
     "value-no-vendor-prefix": true,
     "number-leading-zero": "never",
     "function-url-quotes": "always",
-    "font-weight-notation": "numeric",
+    "font-weight-notation": "named-where-possible",
```
Comment (awebdeveloper) [inline .stylelintrc.json]: y

File: asset/rules/curly.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+ESLintConfigGenApp.rules["curly"] = [
+  {
+    key: "rules.curly",
+    hint: `Enforce consistent brace style for all control statements
+          (<a href='http://eslint.org/docs/rules/curly'>curly</a>)`,
+    variants: [
+      {
+        hint: "Enforce the use of curly braces in all block statements( if, else, for, while, or do )",
```
Comment (awebdeveloper) [inline asset/rules/curly.js]: completely wrong

File: .stylelintrc.json (hunk 2 above)
Comment (deepak-brahmania) [inline .stylelintrc.json]: to exclude -webkit and -moz warning.

File: .stylelintrc.json (hunk 3 above)
Comment (deepak-brahmania) [inline .stylelintrc.json]: To include notation like [bold, normal, bolder] wherever possible

File: asset/css/style.css (hunk 1 above)
Comment (deepak-brahmania) [inline asset/css/style.css]: Reverting back

**Comment by deepak-brahmania** [comment]
screen shot 2017-06-21 at 19 07 34

Rolled back css and included highlights in the comments

File: index.html [hunk 5]
```diff
@@ -1,8 +1,8 @@
----
-layout: default
----
 
 <link href="./asset/css/style.css" rel="stylesheet" type="text/css" />
+<div class="flex header">
```
Comment (awebdeveloper) [inline index.html]: plz remove this

**Comment by deepak-brahmania** [comment]
header tag removed

File: asset/rules/curly.js [hunk 6]
```diff
@@ -20,28 +20,28 @@ ESLintConfigGenApp.rules["curly"] = [
         hint: "Enforce the use of curly braces when multiple statements are present in the block & warn when there is only one statement in the block",
```
Comment (awebdeveloper) [inline asset/rules/curly.js]: these are mui[ple questions, not options of same

**Pull request closed without merging**